src/stacks-hierarchy/Root_TicketAssistantStack/Root_TicketAssistantStack.tsx

```diff
@@ -48,7 +48,9 @@ export const Root_TicketAssistantStack = ({navigation}: Props) => {
                 },
               }
         }
-        rightButton={{type: 'chat'}}
+        rightButton={
+          activeTab !== 5 && activeTab !== 0 ? {type: 'close'} : undefined
```

Comment:
Is there some way to find out how many tabs there is? When just using 5 "magically" in the code like this, it will most likely be forgotten to change if a tab is added or removed.